modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/OrthologAnnotation_conf.pm

```diff
@@ -0,0 +1,185 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+ Bio::EnsEMBL::Production::Pipeline::PipeConfig::OrthologAnnotation_conf;
+
+=head1 DESCRIPTION
+
+=head1 AUTHOR 
+
+ mchakiachvili@ebi.ac.uk
+
+=cut
+
+package Bio::EnsEMBL::Production::Pipeline::PipeConfig::OrthologAnnotation_conf;
+
+use strict;
+use warnings;
+use Bio::EnsEMBL::Hive::Version 2.5;
+use Bio::EnsEMBL::ApiVersion qw/software_version/;
+use base ('Bio::EnsEMBL::Hive::PipeConfig::EnsemblGeneric_conf');
```

Comment:
Better to inherit from Bio::EnsEMBL::Production::Pipeline::PipeConfig::Base_conf. You could then simplify this config by removing the `beekeeper_extra_cmdline_options` and `resource_classes` methods.